scheduler/clients/javascript/lib/userClient.ts

```diff
@@ -117,12 +138,59 @@ export class NettuUserClient extends NettuBaseClient {
     return this.delete<UserResponse>(`/user/${userId}`)
   }
 
-  public freebusy(userId: UUID, req: GetUserFeebusyReq) {
-    return this.get<GetUserFeebusyResponse>(`/user/${userId}/freebusy`, {
-      startTime: req.startTime.toISOString(),
-      endTime: req.endTime.toISOString(),
-      calendarIds: req.calendarIds,
-    })
+  public async freebusy(
+    userId: UUID,
+    req: GetUserFeebusyReq
+  ): Promise<APIResponse<GetUserFeebusyResponse>> {
+    const res = await this.get<GetUserFeebusyResponse>(
+      `/user/${userId}/freebusy`,
+      {
+        startTime: req.startTime.toISOString(),
+        endTime: req.endTime.toISOString(),
+        calendarIds: req.calendarIds?.join(','),
+      }
+    )
+
+    if (!res.data) {
+      return res
+    }
+
+    return {
+      res: res.res,
+      status: res.status,
+      data: {
+        busy: res.data.busy.map(convertInstanceDates),
+      },
+    }
+  }
+
+  public async freebusyMultipleUsers(
+    req: GetMultipleUsersFeebusyReq
+  ): Promise<APIResponse<GetUserFeebusyResponse>> {
+    const res = await this.post<GetUserFeebusyResponse>(
+      '/user/multipleFreebusy',
```

Comment:
I can see why you've done this but I think it's not very RESTful

`/user/freebusy` is more intuitive in my opinion, because my thought process goes like

'okay, to get the freebusy for one user, `/user/{id}/freebusy`, for multiple users, just drop the id.'

The library function can still be called whatever you like.

Reply:
Good point, I admit I wasn't sure about this one. I'll adapt in a bit 👍